File: kattis/testdrive.py
positions = [int(i) for i in input().split()]

diff1 = positions[1] - positions[0]
diff2 = positions[2] - positions[1]

if diff1 * diff2 < 0:
    print("turned")
elif abs(diff1) > abs(diff2):
    print("braked")
elif abs(diff1) < abs(diff2):
    print("accelerated")
else:
    print("cruised")

File: kattis/testdrive.rb
x, y, z = gets.chomp.split(' ').map(&:to_i)
if y - x == z - y
    puts "cruised"
elsif y - x > 0 && z - y < 0 || y - x < 0 && z - y > 0
    puts "turned"
elsif (y - x).abs > (z - y).abs
    puts "braked"
else
    puts "accelerated"
end